### DISCUSSION TITLE
Sentry Logs

### DISCUSSION THREAD

**Discussion by ndegwamartin** [opensrp/fhircore#2663] [Announcements]
**Describe the Issue**
We need to define extra helpful info and metrics that we'd want to log on Sentry. We can track these here.

**Errors**
- [x] Crashes
- [x] Exceptions
- [ ] Login failures

**Performance**

(Not sure if we want to time these, since that will add burden to the app, it could be a default off option.)

- [ ] Sync time
- [ ] Questionnaire load time
- [ ] $apply operation time
- [ ] $evaluate-measure operation time

**Comment by pld** [comment]
I added some related to performance, but not sure we want to do those, or at least if we do not sure we want to default them on

**Comment by ndegwamartin** [comment]
For Log in failures we are able to get such events on Keycloak itself so that any such attempts form any source are logged. This can be enabled for any projects
